ext/rugged/rugged_repo.c

```diff
@@ -2543,6 +2581,33 @@ static VALUE rb_git_repo_cherrypick_commit(int argc, VALUE *argv, VALUE self)
 	return rugged_index_new(rb_cRuggedIndex, self, index);
 }
 
+void rugged_parse_apply_options(git_apply_options *opts, git_apply_location_t *location, VALUE rb_options)
```

Comment:
I see you're passing in the `git_apply_options` here, but it's not being actually filled by any of the fields of the `rb_options` hash.